classes/classes/Items/Consumables/MinotaurBlood.as

```diff
@@ -24,16 +24,7 @@ package classes.Items.Consumables
 		{
 			var tfSource:String = "minotaurBlood";
 			player.slimeFeed();
-			//Changes done
-			changes = 0;
-			//Change limit
-			changeLimit = 1;
-			if (rand(2) === 0) changeLimit++;
-			if (rand(3) === 0) changeLimit++;
-			if (rand(3) === 0) changeLimit++;
-			if (player.findPerk(PerkLib.HistoryAlchemist) >= 0) changeLimit++;
-			if (player.findPerk(PerkLib.TransformationResistance) >= 0) changeLimit--;
-			if (changeLimit === 1) changeLimit = 2;
```

Comment:
Maybe an oversight from the original author: fail all rolls and have the `TransformationResistance` Perk, the `changeLimit` would be 0 and this line would not be executed.

Reply:
Hence, why this transformation is using the third parameter.
```as3
mutations.initTransformation([2, 3, 3], 1, 2);
```

Reply:
@brrritssocold:
Are you still requesting changes here?